plugins/org.jboss.tools.vpe/src/org/jboss/tools/vpe/editor/VpeEditorPart.java

```diff
@@ -1171,7 +1171,7 @@ public void updatePartAccordingToPreferences() {
 	
 	public void updateSelectionBar(boolean isSelectionBarVisible) {
 		if (selectionBar != null) {
-			selectionBar.setVisible(isSelectionBarVisible);
+			selectionBar.setRealBarVisible(isSelectionBarVisible);
```

Comment:
The class also has several commented lines selectionBar.setVisible. Maybe they should be refactored too, just in case that someone later decidess to uncomment them.